### PR TITLE
cli: remove duplicated code

### PR DESCRIPTION
**Pull request by shenyiling**

### CODE REVIEW

File: packages/cli/src/bin/pipcook.ts [hunk 1]
```diff
@@ -38,7 +38,7 @@ import devPlugin from '../actions/dev-plugin';
     versionStr.push(`Pipcook Daemon  v${daemonPkg.version} ${constants.PIPCOOK_DAEMON_SRC}`);
   }
   if (await pathExists(boardPath)) {
-    const boardPkg = require(join(constants.PIPCOOK_BOARD_SRC, 'package.json'));
+    const boardPkg = require(boardPath);
```
Comment (FeelyChau) [inline packages/cli/src/bin/pipcook.ts:41]:
```suggestion
    const boardPkg = await ReadJson(boardPath);
```

Comment (shenyiling) [reply packages/cli/src/bin/pipcook.ts:41]: what mean of it?why is it different from [require(daemonPath)](https://github.com/alibaba/pipcook/blob/master/packages/cli/src/bin/pipcook.ts#L37)?

Comment (FeelyChau) [reply packages/cli/src/bin/pipcook.ts:41]: `require` is a sync function, `ReadJson` is async.

**Comment by FeelyChau** [comment]
@shenyiling We should use asynchronous methods whenever possible to ensure concurrency, so the function `require` should be instead with `readJson`.

**Pull request merged**